### PR TITLE
[Docs] Adds documentation for PostCode validator

### PR DESCRIPTION
**Pull request by froschdesign**

### CODE REVIEW

File: doc/book/validators.md [hunk 1]
```diff
@@ -241,3 +241,94 @@ instantiation. Please update your code to refer to the `IsInt` class instead.
...
+$validator = new Zend\I18n\Validator\PostCode('de_AT');
+$validator->setFormat('AT-\d{5}');
+```
+
+> ### Conventions for self defined formats
+>
+> When using self defined formats, you should omit the regex delimiters and
+> anchors (`'/^'` and  `'$/'`). They are attached automatically.
+>
+> You should also be aware that postcode values are always be validated in a
```
Comment (adamlundrigan) [inline doc/book/validators.md]: Should be either "are always validated" or "will always be validated"

File: doc/book/validators.md [hunk 2]
```diff
@@ -241,3 +241,94 @@ instantiation. Please update your code to refer to the `IsInt` class instead.
...
+> additional characters when they are not covered by the format.
+
+### Constructor options
+
+At its most basic, you may pass a string representing a fully qualified locale
+to the constructor of `Zend\I18n\Validator\PostCode`.
+
+```php
+$validator = new Zend\I18n\Validator\PostCode('de_AT');
+$validator = new Zend\I18n\Validator\PostCode($locale);
```
Comment (adamlundrigan) [inline doc/book/validators.md]: I would remove this line as it serves no purpose here:

> $validator = new Zend\I18n\Validator\PostCode($locale);

**Review by adamlundrigan — APPROVED** [review]
Thanks!  I've made two suggestions for modifications to the original text...may as well include the fixes into this PR while this section of text is in the spotlight ;)

File: doc/book/validators.md (hunk 2 above)
Comment (froschdesign) [reply doc/book/validators.md]: :+1:

File: doc/book/validators.md (hunk 1 above)
Comment (froschdesign) [reply doc/book/validators.md]: Right. :+1:

**Review by adamlundrigan — APPROVED** [review]

**Comment by froschdesign** [comment]
@weierophinney 
Documentation was not updated. The same for zend-validator.

- `PostCode` validator is missing: https://docs.zendframework.com/zend-i18n/validators/
- `PostCode` validator is stil present: https://docs.zendframework.com/zend-validator/validators/post-code/

[Travis says nothing – not build, no output.](https://travis-ci.org/zendframework/zend-i18n/jobs/233319049#L408)

**Comment by weierophinney** [comment]
The oauth token we were using to push documentation was revoked due to the
recent Travis security issue; I need to generate a new one and reconfigure
all repositories.

**Comment by froschdesign** [comment]
@weierophinney 
Good to know.
Thanks for the quick response!

**Pull request merged**